src/frontend/app/app-authenticated/app-authenticated.tsx

```diff
@@ -28,6 +29,7 @@ const AppAuthenticated: React.FC = () => {
             <Route path={routes.CHANGE_REQUESTS} component={ChangeRequests} />
             <Route path={routes.SETTINGS} component={Settings} />
             <Route path={routes.HELP} component={HelpPage} />
+            <Route path={routes.TEAMS} component={TeamsPage} />
```

Comment:
Personally just think it should go right before settings just cuz those are single sorta static pages

Reply:
do you mean on the sidebar?